.eslintrc

```diff
@@ -35,6 +35,7 @@
     },
     "rules": {
         "camelcase": ["off"],
+        "react/jsx-boolean-value": ["off"],
```

Comment:
It would probably be worthwhile to switch to BRUI's eslint config in the near future.

Reply:
Yea. I think I added a lot of UI elements config in here but it seems like some of it may not have made it into annotations